python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -108,7 +108,46 @@ def _apply_nms(orig_prediction: dict, iou_thresh: float = 0.5):
     return nms_prediction
 
 
-def _wrap_image_model(model, examples, model_task, is_function, number_of_classes=None):
+def _process_mlflow_detections_to_raw_detections(image_detections, label_dict: dict, image_size: tuple):
+    """Process AutoML mlflow object detections.
+
+    :param image_detections: AutoML mlflow detections
+    :type image_detections: list
+    :param label_dict: Label dictionary mapping class names to an index
+    :type label_dict: dict
+    :param image_size: Image size
+    :type image_size: tuple(int, int)
+    :return: Raw detections
+    :rtype: dict
+    """
+
+    x, y = image_size
+
+    boxes = []
+    scores = []
+    labels = []
+    for detection in image_detections:
+        label = detection['label']
+        score = detection['score']
+        box = detection["box"]
+
+        ymin, xmin, ymax, xmax = (
```

Comment:
Nit: maybe some comments describing what computations/scaling is being done to coordinates here (and perhaps also in the docstring if there are assumptions about the values of the inputs).